cirq-google/cirq_google/ops/fsim_via_model_tag.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A class that can be used to denote FSim gate implementation using polynomial model."""
+from typing import Any, Dict
+
+import cirq
+
+
+class FSimViaModelTag:
```

Comment:
nit: Would it make sense to be more specific in the name, e.g. `FSimViaPolyModelTag`?

Reply:
`FSimViaModel` is also the name used in the pyle. (I wish I thought twice on this name). But maybe without `Poly` is preferred -- in case we have other modeling approach to do the implementation in the future.